### PR TITLE
Fix shellcheck round2

### PR DESCRIPTION
**Pull request by LanderU**

### CODE REVIEW

**Comment by LanderU** [comment]
@shuhaowu, sorry, I found more warnings, I'll prepare the CI PR now :)

File: focal-rt-ros2/vars.sh [hunk 1]
```diff
@@ -25,32 +27,38 @@ export PINNED_CPU_FREQUENCY=1500000
...
 
 # The image url to download and customize.
+# shellcheck disable=SC2034
 IMAGE_URL="https://cdimage.ubuntu.com/releases/20.04.3/release/ubuntu-20.04.3-preinstalled-server-arm64+raspi.img.xz"
 
 # The partition number of the rootfs. Partition number starts at 0.
 # TODO: This should always be the last partition, otherwise resize won't work.
 # Need the resize because the default image has a very small paritition whereas
 # ROS is big and we need to add some temporary files.
+# shellcheck disable=SC2034
```
Comment (shuhaowu) [inline focal-rt-ros2/vars.sh:37]: Is it possible to disable these for this entire file, or maybe even globally for this project? The whole point of this file is to be sourced.

Comment (LanderU) [reply focal-rt-ros2/vars.sh]: @shuhaowu, good point, done!

**Review by shuhaowu — APPROVED** [review]

**Comment by shuhaowu** [comment]
Thanks!

**Pull request merged**